diagnostic.md

```diff
@@ -21,13 +21,21 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+# curl --include --request GET "http://www.example.com/donuts"
+
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = /* function (id) {
+  return $.ajax({
+    url: app.host + '/donuts/',
+    method: 'GET',
+  });
+};
+
+ */
 ```
```

Comment:
why did you comment this out? without the /* *\ it would be almost spot on.

app.host assumes you've set those variables elsewhere. Looking for:

```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};

```